### PR TITLE
Spacing issues

### PR DESCRIPTION
**Pull request by inikolova**
Fixes https://github.com/telerik/kendo-themes-private/issues/251

### CODE REVIEW

**Review by epetrow — COMMENTED** [review]
Looks good overall, just a few comments:

1. Add a milestone
2. I noticed that we still have some "hardcored" values for some of the variables, which we could easily replace with `k-spacing` values although it might not be part of this effort
3. I might be missing something, but some of the generated visual previews seem odd -- for example the ChartWizard is present as a difference for some themes, but for others it is not. Is that intentional? Custom Card in Fluent and Material as well -- there does not seem to be a visual difference, but could you check ( maybe the branch is stale ) ?

File: tests/_output/bootstrap/timeline/timeline-horizontal.png
Comment (Juveniel) [inline tests/_output/bootstrap/timeline/timeline-horizontal.png:1]: the track is broken

File: tests/_output/material/timeline/timeline-horizontal.png
Comment (epetrow) [inline tests/_output/material/timeline/timeline-horizontal.png:1]: The box-shadow of the second event looks like it's cut off now.

**Review by epetrow — APPROVED** [review]
The inconsistencies in the generated visual previews seem browser related, everything else seems okay to me.

File: packages/fluent/scss/timeline/_layout.scss [hunk 1]
```diff
@@ -377,7 +377,6 @@
         }
 
         .k-timeline-scrollable-wrap {
-            padding-bottom: var( --kendo-timeline-track-wrap-padding-bottom, #{ $kendo-timeline-track-wrap-padding-bottom } );
```
[the remark this answers is not in the document]
Comment (inikolova) [reply packages/fluent/scss/timeline/_layout.scss:380]: This padding moves the timeline track and causes misalignment, so in my opinion it is not needed. However, I didn't remove the variable itself, as it participates in other variables calculations

[the remark this answers is not in the document]
Comment (inikolova) [reply packages/fluent/scss/timeline/_layout.scss]: Update -> after an offline discussion with @imasyuko we decided to skip the current Timeline-spacing-related changes and include them as part of a general component's enhancement.

**Pull request merged**